### PR TITLE
improves crs explanation in ch2

### PR DESCRIPTION
**Pull request by Nowosad**

### CODE REVIEW

File: 02-spatial-data.Rmd [hunk 1]
```diff
@@ -1116,12 +1121,9 @@ projection(new_raster3) = "+proj=utm +zone=12 +ellps=GRS80 +towgs84=0,0,0,0,0,0,
                             +units=m +no_defs" # set CRS
 ```
 
-```{r raster-crs, echo=FALSE, fig.cap="Examples of geographic (WGS 84; left) and projected (NAD83 / UTM zone 12N; right) coordinate systems for raster data.", message=FALSE, fig.asp=0.56, fig.scap="Examples of geographic and projected CRSs (raster data)."}
-# source("https://github.com/Robinlovelace/geocompr/raw/master/code/02-raster-crs.R")
-knitr::include_graphics("figures/02_raster_crs.png")
-```
-
-We will expand on CRSs and how to project from one CRS to another in much more detail in Chapter \@ref(reproj-geo-data).
+Importantly, the `st_crs()` and `projection()` functions do not alter coordinates' values or geometries.
```
Comment (Robinlovelace) [inline 02-spatial-data.Rmd:1124]: Important distinction :+1:

File: 02-spatial-data.Rmd [hunk 2]
```diff
@@ -1116,12 +1121,9 @@ projection(new_raster3) = "+proj=utm +zone=12 +ellps=GRS80 +towgs84=0,0,0,0,0,0,
...
+Their role is only to set a metadata information about the object CRS.
+We will expand on CRSs and explain how to project from one CRS to another in Chapter \@ref(reproj-geo-data).
```
Comment (Robinlovelace) [inline 02-spatial-data.Rmd:1126]: This is a great cross reference, way better.

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Big thumbs up from me. Can you review/merge as appropriate @jannes-m ? Thanks!

**Comment by jannes-m** [comment]
Thanks a lot @Nowosad, good work! I know the text already tells the reader that the commands `projection()`  and `st_crs()` only set the CRS without doing any transformations. Still, to avoid any confusion it might be a good idea to tell the reader that  changing the CRS of `new_raster3` without applying the corresponding transformations, makes `new_raster3` basically useless. But if you think that it is already clear enough, leave it as is.

**Comment by Nowosad** [comment]
:+1

**Pull request merged**